Modules/Control/Control.py

```diff
@@ -35,14 +35,14 @@ def set_control(self, mode=1, offset=[0,0]):
             field = self.magnetometer.get_axes()
             timer = (self.time-time.time()) % (90 * 60)
             torque = get_angle_vector(get_angle(timer))
-            z = (field[1]*torque[0] - filed[0]*torque[1]) / (field[2]*torque[1] - field[1]*torque[2])
+            z = (field[1]*torque[0] - filed[0]*torque[1]) / (field[2] * torque[1] - field[1] * torque[2])
```

Comment:
> "Whom do you show half work to?" - Wise elder Orna Segal

Reply:
It is a quote that orna was mistaken in

Reply:
Well still you have spaced only a half of the line xx